test/http.spec.js

```diff
@@ -0,0 +1,29 @@
+import nock from 'nock';
+import path from 'path';
+import downloadFile from '../src/download-file';
+import fs from 'fs';
+import expect from 'expect';
+
+nock('http://local.foo.com')
```

Comment:
The API is very good actually. Are you suggesting to create some kind of helpers for the expects? I think we can just use the nock configuration before the tests.